docs/application/design/tv/ui-components/presentation-views.md

```diff
@@ -4,6 +4,87 @@ Presentation views display content in different ways.
 
 The following UI components are presentation views:
 
--   [Grid view](presentation-views/grid-view.md)
--   [List view](presentation-views/list-view.md)
--   [Progress](presentation-views/progress.md)
+-   [Grid view](#grid-view)
+-   [List view](#list-view)
+-   [Progress](#progress)
+
+## Grid View
+
+A grid view is used to display graphical content. It consists of grid items and optional labels. The grid items can optionally have additional information.
+
+A grid item can be in either of the following states:
+
+-   Normal
+-   Focused
+-   Selected
+-   Disabled
+
+The following guidelines apply to grid items:
+
+-   The grid item displays only graphical items, such as icons and images.
+-   If the label length exceeds the text area, the excess characters are replaced with an ellipsis (...).
+-   If available, additional information is shown over the grid item.
+
+![grid view](media/uc_02_1_ui_grid_view_re-850x478.png)<br>
+*Grid view*
+
+## List View
+
+A list view is used to show multiple text-based items that belong to the same category.
+
+A list item can be in either of the following states:
+
+-   Normal
+-   Focused
+-   Selected
+-   Disabled
+
+The following guidelines apply to list items:
+
+-   List items can contain a label, an icon, or both.
+-   The label is limited to 1 line of text.
+-   If the label length exceeds the text area, the excess characters are replaced with an ellipsis (...).
+
+![list view](media/uc_02_2_ui_list_view-850x949.png)<br>
+*List view*
+
+## Progress
+
+Progress components provide information about the current state of ongoing tasks.
+
+There are 2 types of progress components:
+
+- [Activity indicator](#Activity-Indicator)
```

Comment:
Suggestion : It looks like `Activity indicator` and `Progress indicator`  sections are quite short. Isn't it better to remove these links?

Reply:
I deleted them.